test/terra/states/test_aer_statevector.py

```diff
@@ -291,6 +295,108 @@ def test_diagonal(self):
         psi = AerStatevector.from_instruction(circuit)
         self.assertEqual(psi, target)
 
+    def test_x(self):
+        """Test x"""
+        circuit = QuantumCircuit(3)
+        circuit.h(0)
+        circuit.x(1)
+        target = AerStatevector.from_label("000").evolve(Operator(circuit))
+        psi = AerStatevector.from_instruction(circuit)
+        self.assertEqual(psi, target)
+
```

Comment:
Are these tests from a different PR, or are they still needed here?

Reply:
Thanks. I wrongly merged another PR to this test. I will fix it soon.